### DISCUSSION TITLE
Kimai Plugin Error

### DISCUSSION THREAD

**Discussion by DustyArmstrong** [kimai/kimai#2920] [On-Premise: Installation & Updates] [answered]
Hi again, 

I got the expenses plugin, but I'm not having great luck with installation. The plugin is detected by Kimai but my logs have the following; 

`Uncaught PHP Exception Doctrine\DBAL\Exception\TableNotFoundException: "An exception occurred while executing 'SELECT COUNT(*) FROM kimai2_expense_category t0 WHERE t0.visible = ?' with params [1]:  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'kimai.kimai2_expense_category' doesn't exist" at /opt/kimai/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 61`

It seems like it's something to do with the SQL table? 

I have put the Plugin (with Docker) into `/opt/kimai/var/plugins/ExpensesBundle`. I appreciate that you don't directly support Docker as a way to run Kimai.

**Comment by DustyArmstrong** [comment, marked as answer]
I'm stupid, I forgot to run:

`bin/console kimai:bundle:expenses:install`

**Reply by kevinpapst** [reply]
Correct 😁 and if you sent me the receipt, I will invite you to the private repository so we can discuss such issues there.

**Reply by DustyArmstrong** [reply]
Thanks, how best to send it to you?
